docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -62,10 +66,9 @@ storage systems:
 * [](/object-storage/file-system-azure)
```

Comment:
get rid of the link to object storage in the list and we also need to fix the sentence before the list